modulefiles/gfsutils_common.lua

```diff
@@ -5,7 +5,7 @@ Load common modules to build GFS utilities on all machines
 local netcdf_c_ver=os.getenv("netcdf_c_ver") or "4.9.2"
 local netcdf_fortran_ver=os.getenv("netcdf_fortran_ver") or "4.6.1"
 
-local bufr_ver=os.getenv("bufr_ver") or "11.7.0"
+local bufr_ver=os.getenv("bufr_ver") or "12.0.1"
```

Comment:
Will this work on on-prem platforms if 12.0.1 is not available in spack-stack 1.6.0 unified env

Reply:
I'll do check to see if bufr 11.7.0 can be loaded on AWS.